<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="mkabany-10" name="org.example.runners.TestRunner" tests="16" failures="1" timestamp="2022-06-12T23:28:46 EET" time="174.194" errors="1">
  <testcase name="runScenario" time="14.694" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="19.041" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="14.048" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="11.107" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.805" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.603" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="8.518" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.879" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="9.716" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.975" classname="org.example.runners.TestRunner">
    <failure type="java.lang.AssertionError" message="expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at org.example.definitions.SliderStepDefinition.checkCurrentUrl(SliderStepDefinition.java:27)
at ✽.user is directed to Nokia Lumia Page(file:///E:/IntelliJ/fwd-selenium-framework/src/main/resources/features/Slider.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="8.177" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="14.039" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.144" classname="org.example.runners.TestRunner">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
at java.util.ArrayList.rangeCheck(ArrayList.java:659)
at java.util.ArrayList.get(ArrayList.java:435)
at org.example.definitions.FollowUsStepDefinition.checkRssCurrentUrl(FollowUsStepDefinition.java:52)
at ✽.user is directed to a new tab with rss website(file:///E:/IntelliJ/fwd-selenium-framework/src/main/resources/features/FollowUs.feature:14)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="14.852" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="9.838" classname="org.example.runners.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="11.758" classname="org.example.runners.TestRunner"/>
  <system-out/>
</testsuite> <!-- org.example.runners.TestRunner -->
